src/gui/relationships.ts

```diff
@@ -133,13 +132,13 @@ export class GuiRelationships extends GuiSubscreen {
 			const allow = e.memberNumber === Player.MemberNumber ? this.relationshipsData.access_modify_self : this.relationshipsData.access_modify_others;
 
 			// Relationships list
-			MainCanvas.strokeRect(130, Y, 700, 64);
+			DrawEmptyRect(130, Y, 700, 64, "white");
 			const name = `${e.name === null ? "[unknown name]" : e.name} (${e.memberNumber})`;
 			DrawTextFit(name, 140, Y + 34, 680, "Black");
 
 			DrawCheckbox(880, Y, 64, 64, "", e.enforced, !allow);
 
-			MainCanvas.strokeRect(994, Y, 500, 64);
+			DrawEmptyRect(994, Y, 500, 64, "white");
```

Comment:
The default stroke color is `Black` and width should be 2 here.
```suggestion
			DrawEmptyRect(994, Y, 500, 64, "Black", 2);
```